gallery_app/lib/src/widgets/box_shape.dart

```diff
@@ -17,9 +17,10 @@ class _XBoxShapeState extends State<XBoxShape> {
       child: Container(
         width: 400,
         height: 100,
-        decoration: BoxDecoration(
-            color: AppColors.slate_50,
-            border: Border.all(color: AppColors.gray_400, width: 1)),
+        decoration: const BoxDecoration(
+          color: AppColors.slate_50,
+          // border: Border.all(color: AppColors.gray_400, width: 1)
```

Comment:
Remove useless code

Reply:
Ummm. Conflict file is gallery_app/lib/src/features/button/view/button_view.dart. In this feature, I only used it to display the UI of XBoxShape... At dev branch it's the button feature